packages/components/psammead-play-button/src/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,431 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`PlayButton should render audio correctly without duration details 1`] = `
+.c3 {
+  vertical-align: middle;
+  margin: 0 0.25rem;
+  color: #222222;
+  fill: currentColor;
+  width: 0.8125rem;
+  height: 0.75rem;
+}
+
+.c1 {
+  -webkit-clip-path: inset(100%);
+  clip-path: inset(100%);
+  -webkit-clip: rect(1px,1px,1px,1px);
+  clip: rect(1px,1px,1px,1px);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  width: 1px;
+  margin: 0;
+}
+
+.c0 {
+  background-color: #222222;
+  border: none;
+  color: #FFFFFF;
+  cursor: pointer;
+  display: block;
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 700;
+  font-style: normal;
+  font-size: 0.75rem;
+  line-height: 1rem;
+  height: 5rem;
+  padding: 0;
+  -webkit-transition: background-color 300ms;
+  transition: background-color 300ms;
+  width: 5rem;
+}
+
+.c0:hover,
+.c0:focus {
+  background-color: #B80000;
+  -webkit-transition: background-color 300ms;
+  transition: background-color 300ms;
+}
+
+.c2 > svg {
+  fill: #FFFFFF;
+  height: 1.5rem;
+  width: 1.5rem;
+}
+
+<button
+  class="c0"
+>
+  <span
+    class="c1"
+  >
+    Play audio, "Dog barks at cat."
```

Comment:
Just checking.... we wouldn't actually have quotes around the tittle would we (unless the journalist has put them in)?

Reply:
According to the UX a11y notes in Zeplin, the title should be wrapped in quotes. Is this an error or because we're using commas as delimiters and the title might include one?